crates/libcontainer/src/tty.rs

```diff
@@ -233,7 +221,7 @@ mod tests {
         let lis = UnixListener::bind(&socket_path);
         assert!(lis.is_ok());
         let fd = setup_console_socket(testdir.path(), &socket_path, CONSOLE_SOCKET);
-        let status = setup_console(&fd.unwrap());
+        let status = setup_console(fd.unwrap().as_raw_fd());
```

Comment:
This test is failing as `setup_console` is closing the `RawFd` it receives.

Reply:
done, changed to `into_raw_fd`